js/packages/web/src/views/art/index.tsx

```diff
@@ -79,6 +87,31 @@ export const ArtView = () => {
                       <span className="creator-name">
                         {creator.name || shortenAddress(creator.address || '')}
                       </span>
+                      {/* <Button
+                        onClick={async () => {
+                          if(!art.mint) {
+                            return;
+                          }
+                          const mint = new PublicKey(art.mint);
+
+                          const account = accountByMint.get(art.mint);
+                          if(!account) {
```

Comment:
This looks interesting...why keep it commented out

Reply:
little secret